gulpfile.js

```diff
@@ -989,6 +989,7 @@ function devGuideExamplesWatch(shredOptions, postShredAction) {
   // gulp.watch([includePattern, excludePattern], {readDelay: 500}, function (event, done) {
   var ignoreThese = [ '**/node_modules/**', '**/_fragments/**', '**/dist/**', '**/typings/**',
                       '**/dart/.pub/**', '**/dart/build/**', '**/dart/packages/**'];
+  ignoreThese = ignoreThese.concat(_exampleBoilerplateFiles.map((file) => `public/docs/_examples/*/*/${file}`));
```

Comment:
This ignores the boilerplate files as well, which account for roughly 1/4 (500ish out of ~2000) of this glob.